src/engine/strat_engine/filesystem.rs

```diff
@@ -86,6 +86,11 @@ impl StratFilesystem {
     pub fn thin_id(&self) -> ThinDevId {
         self.thin_dev.id()
     }
+
+    /// Tear down the filesystem.
```

Comment:
I don't know much about this area. You are saying that you are pretty sure there is nothing?